src/core/mac/mac.hpp

```diff
@@ -65,13 +65,21 @@ namespace Mac {
  */
 enum
 {
+#ifdef OPEN_THREAD_MAC_NO_MIN_BACKOFF
+    kMinBE                = 0,                     ///< macMinBE (IEEE 802.15.4-2006)
+#else
     kMinBE                = 3,                     ///< macMinBE (IEEE 802.15.4-2006)
+#endif
     kMaxBE                = 6,                     ///< macMaxBE (IEEE 802.15.4-2006)
     kMaxCSMABackoffs      = 4,                     ///< macMaxCSMABackoffs (IEEE 802.15.4-2006)
     kMaxFrameRetries      = 15,                    ///< macMaxFrameRetries (IEEE 802.15.4-2006)
     kUnitBackoffPeriod    = 20,                    ///< Number of symbols (IEEE 802.15.4-2006)
-
+    
+#ifdef OPEN_THREAD_MAC_NO_MIN_BACKOFF
+    kMinBackoff           = 0,                     ///< Minimum backoff (milliseconds).
+#else
     kMinBackoff           = 16,                    ///< Minimum backoff (milliseconds).
```

Comment:
My testing found that this severely reduced the speed of OpenThread (more details [here](http://stackoverflow.com/questions/38793140/are-there-performance-goals-or-requirements-for-openthread)). This define here was my testing of what happens when I reduced the min backoff to 0. Are these values set in stone or could we reduce them?